### ISSUE TITLE
Why no ae2?

### ISSUE THREAD

**Issue by Colonizor48** [TeamAOF/All-of-Fabric-4#93]
Title.

**Comment by dandin87** [comment]
Could you show me the 1.17 Version of applied energistics 2? Because I've fairly certain it does not yet exist....
https://www.curseforge.com/minecraft/mc-mods/applied-energistics-2/files

**Comment by Colonizor48** [comment]
https://github.com/AppliedEnergistics/Applied-Energistics-2/tree/fabric/1.17

**Comment by dandin87** [comment]
Yes, Unreleased.... Has to be on curseforge to be in the pack.....
